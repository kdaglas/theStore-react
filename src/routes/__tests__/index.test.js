import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../../routes/index';


describe('The store', () => {
  it('should reach the store without crushing', () => {
    const component = shallow(<Routes />);
    expect(component).toMatchSnapshot();
  });
});
