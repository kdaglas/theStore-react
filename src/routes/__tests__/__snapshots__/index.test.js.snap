// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The routes page should render all Routes without crushing 1`] = `ShallowWrapper {}`;

exports[`The store should reach the store without crushing 1`] = `ShallowWrapper {}`;
